lib/src/components/NxProgressBar/NxProgressBar.scss

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @use "sass:math";
+
+.nx-progress-bar {
+  display: flex;
+  flex-direction: column;
+  gap: var(--nx-spacing-2x);
+  margin: var(--nx-spacing-6x) 0;
+
+  &--inline-counter {
+    flex-direction: row;
+  }
+
+  &--inline {
+    display: inline-flex;
+    margin: 0 var(--nx-spacing-6x);
```

Comment:
The way these margins are set up causes a minor problem: when in `inline-counter` mode, the counter's right margin is still active, effectively mandating that much extra space at the right end of the component.  This is particularly relevant in full-width mode, where that extra space makes it not-quite-full-width.

My recommendation would be for each of the three second-row elements - the counter, icon, and label - to exclusively use left margin to manage the expected space between them.